code/modules/religion/rites.dm

```diff
@@ -634,3 +634,116 @@
 	if(istype(T))
 		T.atmos_spawn_air("water_vapor=5000;TEMP=255")
 	return ..()
+
+/**** Plant rites ****/
+
+/datum/religion_rites/summon_animals
+	name = "Create Life"
+	desc = "Creates a few animals, this can range from butterflys to giant frogs! Please be careful."
+	ritual_length = 30 SECONDS
+	ritual_invocations = list("Great Mother ...",
+	"... bring us new life ...",
+	"... to join with our nature ...",
+	"... and live amongst us ...")
+	invoke_msg = "... We summon thee, Animals from the Byond!" //might adjust to beyond due to ooc/ic/meta
+	favor_cost = 500
+
+/datum/religion_rites/summon_animals/perform_rite(mob/living/user, atom/religious_tool)
+	var/turf/altar_turf = get_turf(religious_tool)
+	new /obj/effect/temp_visual/bluespace_fissure/long(altar_turf)
+	user.visible_message("<span class'notice'>A tear in reality appears above the altar!</span>")
+	return ..()
+
+/datum/religion_rites/summon_animals/invoke_effect(mob/living/user, atom/religious_tool)
+	..()
+	var/turf/altar_turf = get_turf(religious_tool)
+	for(var/i in 1 to 8)
+		var/mob/living/simple_animal/S = create_random_mob(altar_turf, FRIENDLY_SPAWN)
+		S.faction |= "neutral"
+	playsound(altar_turf, 'sound/ambience/servicebell.ogg', 25, TRUE)
+	if(prob(0.1))
+		playsound(altar_turf, 'sound/effects/bamf.ogg', 100, TRUE)
+		user.visible_message("<span class='boldwarning'>A large form seems to be forcing its way into your reality via the portal you opened! RUN!!!</span>")
```

Comment:
This should probably just be a to_chat, since it will look weird for other people to see "you", and also it's coming from the person who did the spell, and not the location of the leaper

Reply:
Adjusted so it displays from the altar where the leaper spawns, and so it shows who summoned the portal